sycl/include/CL/sycl/detail/os_util.hpp

```diff
@@ -28,9 +28,23 @@
 #endif // _WIN32
 
 #if defined(SYCL_RT_OS_WINDOWS)
+
 #define DLL_LOCAL
+// If SYCL headers are included to build SYCL library then the macro is used
+// to set dllexport attribute for global variables/functions/classes.
+// Otherwise, the macro is used used to set dllimport for the same global
+// variables/functions/classes.
+#if defined(__SYCL_BUILD_SYCL_DLL)
+#define __SYCL_DLL_API __declspec(dllexport)
```

Comment:
Can we avoid DLL in the name? How about __SYCL_EXPORTED

Reply:
Then, may be __SYCL_API ? I don't like the word EXPORT as it is not necessary export it may be defined to import.

Reply:
I gave it another thought...
__SYCL_API gives less info about what that means.
I think __SYCL_EXPORT or __SYCL_EXPORTED is Ok/better.